front/src/common/redux/forgot-password/forgot.password.api.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+export async function forgotPassswordApi (request) {
+    const url = `${process.env.REACT_APP_FORGOT_PASSWORD_ROUTE}`;
+    const headers = { headers: { 'Content-Type': 'application/json' } };
+
+    return axios
+        .post(url, request.data, headers)
+        .then(response => response)
```

Comment:
API function should return only the actual data. Saga should know nothing about any HTTP related data

Reply:
done